tournamentcontrol/competition/tests/test_logo_fields.py

```diff
@@ -0,0 +1,174 @@
+from test_plus import TestCase as TestPlusTestCase
+
+from tournamentcontrol.competition.models import (
+    Club,
+    Competition,
+    Division,
+    Season,
+    Team,
+)
+from tournamentcontrol.competition.tests.factories import (
+    ClubFactory,
+    CompetitionFactory,
+    DivisionFactory,
+    SeasonFactory,
+    TeamFactory,
+)
+from tournamentcontrol.competition.validators import validate_logo_aspect_ratio
+
+
+class LogoFieldTestCase(TestPlusTestCase):
+    """Test that logo fields exist and accept valid file formats."""
+
+    def test_competition_has_logo_fields(self):
+        """Verify Competition model has logo_colour and logo_monochrome fields."""
+        competition = CompetitionFactory()
+        self.assertTrue(hasattr(competition, "logo_colour"))
+        self.assertTrue(hasattr(competition, "logo_monochrome"))
+
+    def test_club_has_logo_fields(self):
+        """Verify Club model has logo_colour and logo_monochrome fields."""
+        club = ClubFactory()
+        self.assertTrue(hasattr(club, "logo_colour"))
+        self.assertTrue(hasattr(club, "logo_monochrome"))
+
+    def test_season_has_logo_fields(self):
+        """Verify Season model has logo_colour and logo_monochrome fields."""
+        season = SeasonFactory()
+        self.assertTrue(hasattr(season, "logo_colour"))
+        self.assertTrue(hasattr(season, "logo_monochrome"))
+
+    def test_division_has_logo_fields(self):
+        """Verify Division model has logo_colour and logo_monochrome fields."""
+        division = DivisionFactory()
+        self.assertTrue(hasattr(division, "logo_colour"))
+        self.assertTrue(hasattr(division, "logo_monochrome"))
+
+    def test_team_has_logo_fields(self):
+        """Verify Team model has logo_colour and logo_monochrome fields."""
+        team = TeamFactory()
+        self.assertTrue(hasattr(team, "logo_colour"))
+        self.assertTrue(hasattr(team, "logo_monochrome"))
+
+    def test_logo_fields_optional(self):
+        """Verify logo fields are optional (blank=True, null=True)."""
+        competition = CompetitionFactory()
+        self.assertFalse(competition.logo_colour.name)
+        self.assertFalse(competition.logo_monochrome.name)
+
+    def test_logo_accepts_png(self):
+        """Verify logo fields accept PNG files."""
+        try:
+            from PIL import Image
+            
+            # Create a simple PNG image in memory
+            img = Image.new("RGB", (100, 100), color="red")
+            img_bytes = io.BytesIO()
+            img.save(img_bytes, format="PNG")
+            img_bytes.seek(0)
+            
+            png_file = SimpleUploadedFile(
+                "test_logo.png", img_bytes.read(), content_type="image/png"
+            )
+            
+            competition = CompetitionFactory(logo_colour=png_file)
+            competition.full_clean()
+            self.assertTrue(competition.logo_colour.name.endswith(".png"))
+        except ImportError:
```

Comment:
Pillow is a project dependency, we will never hit this. Don't test for it, it should raise an error rather than skip.